Templates/layouts/fail_tasks.phtml

```diff
@@ -0,0 +1,44 @@
+<?php echo isset($errMsg) ? $errMsg : '' ?>
```

Comment:
u can add variable declaration ah the first line of template to prevent red highlighting your variable in the file. like
`<?php /** @var $options array */ ?>`